server/utils/dateUtils.test.ts

```diff
@@ -158,4 +158,60 @@ describe('DateFormats', () => {
       expect(differenceInDays).toHaveBeenCalledWith(date1, date2)
     })
   })
+
+  describe('dateAndTimeInputsToUiDate', () => {
+    it('converts a date and time input object to a human readable date', () => {
+      const dateTimeInputs = { 'key-day': '11', 'key-month': '11', 'key-year': '2022' }
+
+      expect(DateFormats.dateAndTimeInputsToUiDate(dateTimeInputs, 'key')).toEqual('11/11/2022')
```

Comment:
probably I'm not understanding, but would it be clearer if day and month were not both `'11'` here?